models/yang/sonic/sonic-acl.yang

```diff
@@ -119,12 +122,15 @@ module sonic-acl {
 				}
 
 				leaf MIRROR_ACTION {
-					type leafref {
-						path "/sms:sonic-mirror-session/sms:MIRROR_SESSION/sms:MIRROR_SESSION_LIST/sms:name";
+					type string {
+						pattern "[^ |:]+" {
```

Comment:
Please redefine the pattern as suggested above to disallow invalid special characters.